### ISSUE TITLE
Fix problem with cycles in PDAG.

### ISSUE THREAD

**Issue by ankurankan** [pgmpy/pgmpy#1300]
Refer to TODOs in DAG.py:PDAG and test_PDAG.py for details.

**Comment by ankurankan** [comment]
@Nuna7 Yes, please add your comment here for reference. I would however suggest not spending time on fixing the issue  to avoid duplicated effort as there is already an open PR for this.

**Comment by Manas-7854** [comment]
Hi @ankurankan, Can i take this up?

**Comment by ankurankan** [comment]
@Manas-7854 This is already done. Sorry we missed to close the issue.
